envs/dev-windows.bat

```diff
@@ -3,3 +3,4 @@ set FLASK_ENV=development
 set SECRET_KEY=dev
 set SPOTIPY_CLIENT_ID="5d141ce6f09c4fbfa12d16ce9e5d40c1"
 set SPOTIPY_CLIENT_SECRET="52fed0d6564a4e6f8e596b78bd1abf62"
+set DISCOVERY_API_KEY="L8vZ5k9U8L8r8r9YstxDbOml0imvZBYmj7bB_tiVQ2GS"
```

Comment:
Remove the doublequotes from this line, I was getting an error about quotes. I guess windows interprets the files differently and maybe spotipy doesn't care

Reply:
Done